logger import missing

```
self = <yapic.session.Session object at 0x7fc75a332c18>, norm_string = 'local'

    def set_normalization(self, norm_string):
        '''
        Set pixel normalization scope.
    
        Parameters
        ----------
        norm_string : string
            For minibatch-wise normalization choose 'local_z_score' or 'local'.
            For global normalization use global_<min>+<max>
            (e.g. 'global_0+255' for 8-bit images and 'global_0+65535' for
            16-bit images).
            Choose 'off' to deactivate.
        '''
    
        if self.data is None:
>           logger.warning(
                'could not set normalizarion to {}. Run make_model() first')
E           NameError: name 'logger' is not defined

yapic/session.py:278: NameError

```

--- a/yapic/session.py
+++ b/yapic/session.py
@@ -8,6 +8,8 @@ from yapic_io.dataset import Dataset
 import keras
 import os
 
+import logging
+logger = logging.getLogger(__name__)
 
 class Session(object):
     '''

--- a/yapic/tests/test_session.py
+++ b/yapic/tests/test_session.py
@@ -56,6 +56,19 @@ class TestSessionMethods(TestCase):
         assert t.data.normalize_mode == 'global'
         assert t.data.global_norm_minmax == (0, 255)
 
+    def test_set_normalization_if_no_data(self):
+        img_path = os.path.join(
+            base_path,
+            '../test_data/shapes/pixels/*')
+        label_path = os.path.join(
+            base_path,
+            '../test_data/shapes/labels.ilp')
+
+        t = Session()
+        # t.load_training_data(img_path, label_path)
+
+        t.set_normalization('local')
+
     def test_load_model(self):
 
 
@@ -136,6 +149,7 @@ class TestEnd2End(TestCase):
         assert 'pixels_2_class_3.tif' in artifacts
 
         shutil.rmtree(savepath)
+
 
     @pytest.mark.slow
     def test_shape_data(self):

--- a/yapic/utils.py
+++ b/yapic/utils.py
@@ -2,8 +2,8 @@ import logging
 import re
 import os
 
-logger = logging.getLogger(os.path.basename(__file__))
-logger.setLevel(logging.INFO)
+logger = logging.getLogger(__name__)
+
 
 
 def handle_augmentation_setting(mbatch, augment):

